docs/MO_DG/prepare_model/Additional_Optimizations.md

```diff
@@ -11,8 +11,41 @@ Model Optimizer performs preprocessing to a model. It is possible to optimize th
 -	**Larger batch size**<br>
 	Notice that the devices like GPU are doing better with larger batch size. While it is possible to set the batch size in the runtime using the OpenVINO Runtime API [ShapeInference feature](../../OV_Runtime_UG/ShapeInference.md).
 
--	**Resulting IR precision**<br>
-The resulting IR precision, for instance, `FP16` or `FP32`, directly affects performance. As CPU now supports `FP16` (while internally upscaling to `FP32` anyway) and because this is the best precision for a GPU target, you may want to always convert models to `FP16`. Notice that this is the only precision that Intel&reg; Movidius&trade; Myriad&trade; 2 and Intel&reg; Myriad&trade; X VPUs support.
+## When to Specify Layout
```

Comment:
In this document we also have a mean / scale options which are also require layouts to be specified. So, should not descriptions of layouts be before other options?

Reply:
It seems correct order to me: firstly we list the optimizations that can be done, secondly, we describe when layouts are needed and after that we describe each optimization.